book-content/chapters/13-modules-scripts-declaration-files.md

```diff
@@ -0,0 +1,988 @@
+In this chapter, we'll be diving deeper into modules. First, we'll look at how TypeScript understands global scope by looking at the distinction between 'modules' and 'scripts'. Second, we'll look at declaration files - `.d.ts files` - and introduce the `declare` keyword.
+
+## Understanding Modules and Scripts
+
+TypeScript has two ways of understanding what a `.ts` file is. It can be treated either as a module, containing imports and exports, or a script, which executes in the global scope.
```

Comment:
😉 
```suggestion
JavaScript has two ways of understanding what a `.js` file is. It can be treated either as a module, containing imports and exports, or a script, which executes in the global scope.
```

Reply:
I know that you mentioned later that modules are available in JS and all - but a few paragraphs at the beginning here make it sound like it's a TS feature and maybe even that somehow it crept to JS or something. I feel like it's not emphasized enough that it's the other way around.